### PR TITLE
Adding new database collections 

### PR DESCRIPTION
**Pull request by WilliamWood01**

### CODE REVIEW

**Comment by WilliamWood01** [comment]
Added repository functions, skipped appointment for now as it seems like it will be out of scope
I will wait until this is merged and then move onto creating routes and service functions as some things can be extracted

File: src/database/cart/cart.txt
Comment (adelinavisan97) [inline src/database/cart/cart.txt:1]: yeah good point.. i think we should be able to do it on the front end but not absolutely sure, will check with Rachel and Abudi

File: src/database/medicine/medicine.repository.ts
Comment (adelinavisan97) [inline src/database/medicine/medicine.repository.ts:9]: yup, no need to add another UI component for this, we'll just populate with postman

File: src/modules/users/models/order.interace.ts
Comment (adelinavisan97) [inline src/modules/users/models/order.interace.ts:11]: if we're using the userId from the mongo record, we can have it as objectId, if we switch to email, then it should be a string yeah. but should be okay to be an object id because that's the type that we're getting from mongo when we fetching the user by email.

**Review by adelinavisan97 — APPROVED** [review]

File: src/modules/users/models/order.interace.ts
Comment (WilliamWood01) [reply src/modules/users/models/order.interace.ts:11]: Yeah this is for the 'request' interface though, and I am not sure the frontend will be passing in the object id when making requests to create an order for a user (I would prefer they did so we might need to edit some of the user functionality to return and store the users object id in the session alongside email)

Comment (adelinavisan97) [reply src/modules/users/models/order.interace.ts:11]: oh i see, yeah let's leave it like this for now and we'll look into how to do it from the front end

**Pull request merged**
